src/ru/seimanga/src/eu/kanade/tachiyomi/extension/ru/seimanga/SeiManga.kt

```diff
@@ -0,0 +1,199 @@
+package eu.kanade.tachiyomi.extension.ru.seimanga
+
+import android.app.Application
+import android.content.SharedPreferences
+import android.widget.Toast
+import androidx.preference.EditTextPreference
+import eu.kanade.tachiyomi.multisrc.grouple.GroupLe
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import okhttp3.Request
+import uy.kohesive.injekt.Injekt
+import uy.kohesive.injekt.api.get
+
+class SeiManga : GroupLe("SeiManga", "https://seimanga.me", "ru") {
+
+    override val id: Long = 6
```

Comment:
```suggestion
```
id should only be overriden if old one needs to be preserved. let it generate automatically